server/index.js

```diff
@@ -11,6 +11,14 @@ app.listen(PORT, () => {
   console.log('listening at port ', PORT);
 });
 
+//Tiers service
 app.use('/tiersIncluded', proxy('localhost:3101'));
```

Comment:
Be forewarned that Leslie has informed us the '`express-http-proxy`' package is 'on the line' and okay for first iteration but 1) should be fully understood 2) may have to be refactored later to indicate understandng.